### PR TITLE
Converts SSblackbox feedback to an associative list

### PR DESCRIPTION
**Pull request by Kapu1178**
### I DID NOT TEST THIS. I DO NOT KNOW DATABASE STUFF. I JUST NOTICED IT WHILE WORKING ON AN UNRELATED PR.
Title. 

## Why It's Good For The Game
Speeds up a hot proc substantially

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Do a pass on all uses of the var, you missed a use in /datum/controller/subsystem/blackbox/Shutdown, so as is the feedback db will not actually log

**Comment by Kapu1178** [comment]
> Do a pass on all uses of the var, you missed a use in /datum/controller/subsystem/blackbox/Shutdown, so as is the feedback db will not actually log

eyyyup. forgot to do that part.

File: code/controllers/subsystem/blackbox.dm [hunk 1]
```diff
@@ -4,7 +4,7 @@ SUBSYSTEM_DEF(blackbox)
 	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
 	init_order = INIT_ORDER_BLACKBOX
 
-	var/list/feedback = list() //list of datum/feedback_variable
+	var/list/feedback_list = list() //list of datum/feedback_variable
```
Comment (LemonInTheDark) [inline code/controllers/subsystem/blackbox.dm:7]: autodoc this, and make the documentation accurate

**Comment by optimumtact** [comment]
>I DID NOT TEST THIS. I DO NOT KNOW DATABASE STUFF. I JUST NOTICED IT WHILE WORKING ON AN UNRELATED PR.

okay then

**Comment by optimumtact** [comment]
mso vouched that this is ok

**Review by MrsTonedOne — APPROVED** [review]
**Old:** (R189104)

`{"name":"/datum/controller/subsystem/blackbox/proc/find_feedback_datum","self":3.585,"total":3.596,"real":3.854,"over":0.033,"calls":813227}`
`{"name":"/datum/controller/subsystem/blackbox/proc/record_feedback","self":3.175,"total":6.783,"real":7.047,"over":0.012,"calls":825695},`

**New:** (R189107)

`{"name":"/datum/controller/subsystem/blackbox/proc/find_feedback_datum","self":0.541,"total":0.548,"real":0.784,"over":0.005,"calls":762460}`
`{"name":"/datum/controller/subsystem/blackbox/proc/record_feedback","self":2.999,"total":3.558,"real":3.808,"over":0.03,"calls":774124}`

There were 74 items in the list when the database saved. they appear to be in the database

**Pull request merged**